### PR TITLE
Import docs from XML build file

### LINKED ISSUE #3199: Missing API reference for System.Runtime.CompilerServices.Unsafe
The .Net Core 2.0 class `System.Runtime.CompilerServices.Unsafe` [does not have any API reference documentation](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.unsafe?view=netcore-2.0). Its methods are listed, but with no explanation about what they do:

### PR DESCRIPTION
**Pull request by BillWagner**
Note that remarks are still missing.

Trying a test build as I add remarks.

Fixes #3199 

[Internal review URL](https://review.docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.unsafe?branch=pr-en-us-3773&view=netcore-2.0)

### CODE REVIEW

**Comment by BillWagner** [comment]
closing and reopening to trigger a new build

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 1]
```diff
@@ -664,11 +712,11 @@
...
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="elementOffset">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="source">The reference to subtract the offset from.</param>
+        <param name="elementOffset">The offset to subtract./param>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: missing closing tag on this line

**Review by mairaw — COMMENTED** [review]
build error probably caused by that line

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 2]
```diff
@@ -38,11 +40,13 @@
         <Parameter Name="elementOffset" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="elementOffset">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="source">TThe reference to add the offset to.</param>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: typo: TThe (3 occurrences)

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 3]
```diff
@@ -123,11 +129,13 @@
...
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="left">TThe first reference to compare.</param>
+        <param name="right">The second reference to compare.</param>
+        <summary>
+          Determines whether the specified references point to the same location.
+        </summary>
+        <returns><c>true</c> if <paramref name="left"/> and <paramref name="right"/> point to the same location; otherwise <c>false</c>.</returns>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: missing comma after otherwise

Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: better to use langword instead of code for these: \<see langword="true" />

**Review by mairaw — CHANGES_REQUESTED** [review]
LGTM. A few things to fix before merging though. 

@joperezr, @AlexGhiondea do you guys need to take a look at this? Given we're mostly using dev comments here, it should be good.

**Review by joperezr — APPROVED** [review]
LGTM other than the already comments from @mairaw

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 4]
```diff
@@ -123,11 +129,13 @@
...
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="left">The first reference to compare.</param>
+        <param name="right">The second reference to compare.</param>
+        <summary>
+          Determines whether the specified references point to the same location.
+        </summary>
+        <returns><<see langword="true" /> if <paramref name="left"/> and <paramref name="right"/> point to the same location; otherwise, <see langword=false" />.</returns>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: there's an extra \< here

**Comment by BillWagner** [comment]
closing and reopening to start a new build.

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 5]
```diff
@@ -637,9 +685,9 @@
       </TypeParameters>
       <Parameters />
       <Docs>
-        <typeparam name="T">To be added.</typeparam>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of object whose size is retrieved.</typeparam>
+        <summary>TReturns the size of an object of the given type parameter.</summary>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: typo: TReturns

File: xml/System.Runtime.CompilerServices/Unsafe.xml [hunk 6]
```diff
@@ -123,11 +129,13 @@
...
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <typeparam name="T">The type of reference.</typeparam>
+        <param name="left">The first reference to compare.</param>
+        <param name="right">The second reference to compare.</param>
+        <summary>
+          Determines whether the specified references point to the same location.
+        </summary>
+        <returns><see langword="true" /> if <paramref name="left"/> and <paramref name="right"/> point to the same location; otherwise, <see langword=false" />.</returns>
```
Comment (mairaw) [inline xml/System.Runtime.CompilerServices/Unsafe.xml]: missing quote for false

Comment (BillWagner) [reply xml/System.Runtime.CompilerServices/Unsafe.xml]: I'm laughing that you find these so quickly. I stared at that line for five minutes and I didn't see it.

**Review by mairaw — APPROVED** [review]

**Pull request merged**